### ISSUE TITLE
Incorrect Cycle for Rings 2/11/2020

### ISSUE THREAD

**Issue by RedDeadDamnation** [jeanropke/RDR2CollectorsMap#838]
Hello,

First off, THANK YOU to the people behind this project!

I apologize if I'm doing this wrong, but I just wanted to let someone know that on 2/11/2020, the Cycle for Lost Jewelry - Rings is apparently wrong.  It states Cycle 3.  I've tried going to the locations for four of them and none have been at the locations stated on the map.  Haven't found any issues with other items today, but haven't tried all of them.

I'm pretty new to this (been playing for ~month and haven't touched on what this map seems capable of) and this has never happened.  All items have always been exactly where the map states (which is absolutely incredible!).

I hope I'm helping by reporting this issue!

Thanks again!

**Comment by RedDeadDamnation** [comment]
Well, now I'm both sorry and confused.  I was just able to obtain the Beaulieux Diamond Ring at the stated location on the map.

The others that I tried (I believe) all required the metal detector, while this one was inside a chimney.  Don't see how that could make a difference, because my metal detector seems to be working fine otherwise.

Perhaps my game is just buggy today.  Not that its creators would ever allows such things.

Here's the link to one of the ones I recently tried to acquire and couldn't (metal detector was dark and silent):
https://jeanropke.github.io/RDR2CollectorsMap/?m=ring_bosque_emerald

Others' experiences would be welcome.

Still just trying to be helpful!
